functional/README.md

```diff
@@ -4,50 +4,93 @@ This functional test suite deploys a fleet cluster using nspawn containers, and
 
 It shares an instance of etcd deployed on the host machine with each of the nspawn containers.
 
-It's recommended to run this in a virtual machine environment on CoreOS (e.g. using coreos-vagrant). The only dependency for the tests not provided on the CoreOS image is `go`.
+It's recommended to run this in a virtual machine environment on CoreOS (e.g. using [Vagrant][test-in-vagrant]).
 
-The caller must do three things before running the tests:
+Since the tests utilize `[systemd-nspawn]`, this needs to be invoked as sudo/root.
 
-1. Ensure an ssh-agent is running and the functional-testing identity is loaded. The `SSH_AUTH_SOCK` environment variable must be set.
+If the tests are aborted partway through, it's currently possible for them to leave residual state as a result of the `systemd-nspawn` operations. This can be cleaned up using the `clean.sh` script.
 
-```
-$ ssh-agent
-$ ssh-add fleet/functional/fixtures/id_rsa
-$ echo $SSH_AUTH_SOCK
-/tmp/ssh-kwmtTOsL7978/agent.7978
-```
-2. Ensure the `FLEETD_BIN` and `FLEETCTL_BIN` environment variables point to the respective fleetd and fleetctl binaries that should be used to drive the actual tests.
+### run tests in vagrant
 
+Vagrant will configure CoreOS instance with one-member etcd2 daemon.
```

Comment:
The recommended way to run the tests is to use the provided Vagrantfile, which will set up a single CoreOS instance with a one-member etcd cluster.